Documentation/gettingstarted/encryption.rst

```diff
@@ -59,16 +59,32 @@ Deploy Cilium release via Helm:
       --set global.encryption.enabled=true \\
       --set global.encryption.nodeEncryption=false
 
+These options can be provided along with other options, such as when deploying
+to GKE, with VXLAN tunneling:
+
+.. parsed-literal::
+
+    helm install cilium |CHART_RELEASE| \\
+      --namespace cilium \
+      --set global.nodeinit.enabled=true \
+      --set nodeinit.reconfigureKubelet=true \
+      --set nodeinit.removeCbrBridge=true \
+      --set global.cni.binPath=/home/kubernetes/bin \
+      --set global.tunnel=vxlan \
+      --set global.encryption.enabled=true \
+      --set global.encryption.nodeEncryption=false
+
```

Comment:
Its probably preferred to run GKE with direct routing but having an example is nice. We can add a direct routing example later.

Reply:
Could we still add one as follow-up, so we don't loose track of it? Thx